IoTTemperatureTracking/no_hardware.js

```diff
@@ -16,12 +14,12 @@ function generateData(stitchClient){
   console.log(data);
 
   // Simulate write to MongoDB every 2 minutes
-  stitchClient.executeFunction("Imp_Write", data).then(
+  stitchClient.callFunction("Imp_Write", data).then(
     setTimeout(() => generateData(stitchClient), 120000)
   );
 }
 
 // Use the API Key to load data
-stitchClientPromise.then(stitchClient => {
-  return stitchClient.authenticate("apiKey", "<STITCH API KEY>").then(() => generateData(stitchClient));
-})
+const stitchClient = Stitch.initializeDefaultAppClient("<STITCH API KEY>");
```

Comment:
this is not going to work. You have to initialize the app client with the app ID, and then authenticate using `stitchClient.auth.loginWithCredential(new ServerApiKeyCredential("<STITCH API KEY>")`.

Also, `Stitch.initializeDefaultAppClient` will need to be `stitch.Stitch.initializeDefaultAppClient` if you are including the SDK with `require`.

Reply:
If possible, it will be better to use ES6 import syntax since `require` is not standard JS.